Makefile

```diff
@@ -128,7 +128,7 @@ tidy:
 checktidy:
 	./etc/checktidy.sh
 
```

Comment:
Can you add `libflux/scanner.c` to `GENERATED_TARGETS`?  This way it will get deleted when `make cleangenerate` is run.

Reply:
Making this change caused CI to fail, reverted back to original solution purposed by @nathanielc